moj-node/cypress/e2e/homepage.spec.js

```diff
@@ -9,15 +9,15 @@ describe('Homepage', () => {
       it('contains the correct items in the navigation', () => {
         cy.get(cssNav)
           .should('contain', 'Home')
-          .should('contain', 'Working in Berwyn')
+          .should('contain', 'Education and work')
           .should('contain', 'Getting out');
       });
 
       it(`navigates to the 'Working in Berwyn' page`, () => {
```

Comment:
Probably also worth updating the test title here?

Reply:
Done